contracts/core/Allo.sol

```diff
@@ -419,7 +530,7 @@ contract Allo is IAllo, Native, Transfer, Initializable, Ownable, AccessControl
         pools[_poolId].strategy.allocate{value: msg.value}(_data, msg.sender);
     }
 
-    /// @notice Deduct the fee and transfers the amount to the distribution strategy
+    /// @notice Deducts the fee and transfers the amount to the distribution strategy
```

Comment:
`@dev Deducts the fee and transfers the amount to the distribution strategy`
`@notice Fund the pool ..`